src/eredis_cluster_monitor.erl

```diff
@@ -118,21 +118,32 @@ get_cluster_slots_from_single_node(Node) ->
 
 -spec parse_cluster_slots([[bitstring() | [bitstring()]]]) -> [#slots_map{}].
 parse_cluster_slots(ClusterInfo) ->
-    Length = erlang:length(ClusterInfo),
-    ClusterInfoI = lists:zip(ClusterInfo,lists:seq(1,Length)),
-    [
-        #slots_map{
-            index = Index,
-            start_slot = binary_to_integer(StartSlot),
-            end_slot = binary_to_integer(EndSlot),
-            node = #node{
-                address = binary_to_list(Address),
-                port = binary_to_integer(Port)
-            }
+    parse_cluster_slots(ClusterInfo, 1, []).
+
+%% redis 3.0.0 - 3.2.0 format
+parse_cluster_slots([[StartSlot, EndSlot | [[Address, Port] | _]] | T], Index, Acc) ->
+    NewAcc = [get_slots_map(Index, StartSlot, EndSlot, Address, Port) | Acc],
+    parse_cluster_slots(T, Index+1, NewAcc);
+
+%% redis 3.2.0 format
+parse_cluster_slots([[StartSlot, EndSlot | [[Address, Port, _] | _]] | T], Index, Acc) ->
```

Comment:
for this, I think it might be better if we reduce the number of function signature and we just have one function:

parse_cluster_slots([[StartSlot, EndSlot | [[Address, Port | _] | _]] | T], Index, Acc)

it will be compatible both for redis 3.2.0 and 3.0.0. (We can save 3 lines of code ! :)